run.sh

```diff
@@ -45,6 +45,15 @@ else
   npm i -g yo generator-react-bootstrap > /dev/null 2>&1
 fi
 
+yo --version
+
+if [ $? -eq 0 ]; then
+  echo YEOMAN installed
+else
+  echo YEOMAN not installed, installing now
```

Comment:
"Yeoman not installed, installing now..."

Add the ... and uppercase is kind of ugly, use "Yeoman" instead of YEOMAN